### PR TITLE
post_generation dependency fixed

### PR DESCRIPTION
**Pull request by olegpidsadnyi**

### CODE REVIEW

File: README.rst [hunk 1]
```diff
@@ -25,9 +25,121 @@ Install pytest-factoryboy
...
+
+
+    register(Author)
+
+
+    def test_model_fixture(author):
+        assert author.name == "Charles Dickens"
+
+
+Model fixtures can be registered with specif names. For example if you address instances of some collection
```
Comment (bubenkoff) [inline README.rst]: typo: specif
For example<comma>

File: README.rst [hunk 2]
```diff
@@ -25,9 +25,121 @@ Install pytest-factoryboy
...
+
+    class AuthorFactory(Factory):
+
+        class Meta:
+            model = Author
+
+        name = "Charles Dickens"
+
+
+    register(Author)
```
Comment (bubenkoff) [inline README.rst]: use decorator variant for such simple example

File: README.rst [hunk 3]
```diff
@@ -25,9 +25,121 @@ Install pytest-factoryboy
...
+    register(AuthorFactory) # author
+    register(AuthorFactory, name="second_author") # second_author
+
+    register(BookFactory, name="other_book")  # other_book, book of another author
+
+    @pytest.fixture
+    def other_book__author(second_author):
+        """Make the relation of the second_book to another (second) author."""
+        return second_author
+
```
Comment (bubenkoff) [inline README.rst]: i'd illustrate this example with the small diagram

File: README.rst [hunk 4]
```diff
@@ -25,9 +25,121 @@ Install pytest-factoryboy
...
+
+
+Attribute Fixture
```
Comment (bubenkoff) [inline README.rst]: Attributes are fixtures

File: README.rst [hunk 5]
```diff
@@ -25,9 +25,121 @@ Install pytest-factoryboy
...
+
+Sub-factory attribute points to the model fixture of the sub-factory.
+Attributes of sub-factories are injected as dependencies to the model fixture and can be overridden in
+the parametrization.
+
+Related Factory
+---------------
+
+Related factory attribute points to the model fixture of the related factory.
+Attributes of related factories are injected as dependencies to the model fixture and can be overridden in
```
Comment (bubenkoff) [inline README.rst]: have a link to the docs about override: http://pytest.org/latest/fixture.html#override-a-fixture-with-direct-test-parametrization

File: pytest_factoryboy/fixture.py [hunk 6]
```diff
@@ -152,5 +165,5 @@ def get_caller_module(depth=2):
     frame = sys._getframe(depth)
     module = inspect.getmodule(frame)
     if module is None:
-        return get_caller_module(depth=depth)
+        return get_caller_module(depth=depth)  # pragma: no cover
```
Comment (bubenkoff) [inline pytest_factoryboy/fixture.py]: mention when it happens

File: README.rst (hunk 3 above)
Comment (olegpidsadnyi) [inline README.rst]: I don't think it is important ATM

**Comment by bubenkoff** [comment]
please rebase to have single commit

**Pull request merged**